### PR TITLE
Uncomment abstract method.

### PR DESCRIPTION
**Pull request by cdata**
This should be safe now, with 0.9

### CODE REVIEW

File: iron-validator-behavior.html [hunk 1]
```diff
@@ -52,11 +52,8 @@
      *
      * @method validate
```
Comment (morethanreal) [inline iron-validator-behavior.html:53]: Does that mean I can also remove this annotation?

**Comment by morethanreal** [comment]
lgtm

File: iron-validator-behavior.html (hunk 1 above)
Comment (cdata) [inline iron-validator-behavior.html:53]: Maybe? @jklein24 suggested that annotations for methods were still a good thing to have.

Comment (jklein24) [inline iron-validator-behavior.html:53]: The compiler doesn't use @method annotations at all, so unless it's useful for the doc generator, feel free to get rid of them.

**Pull request merged**